src/report_reader.cpp

```diff
@@ -112,6 +112,10 @@ auto SpikeReader::openPopulation(const std::string& populationName) const -> con
     return populations_.at(populationName);
 }
 
+std::tuple<double, double> SpikeReader::Population::getTimes() const {
```

Comment:
can you add tests and python bindings for this?

Reply:
done